src/skills.cpp

```diff
@@ -857,7 +857,7 @@ void init_skills(void) {
            DAM_UNDEFINED, SKILL_SPHERE_HEALING, 5, true, nullptr);
 
     spello(SPELL_GROUP_RECALL, "group recall", 50, 30, 2, POS_STANDING, true, TAR_IGNORE, false, MAG_GROUP, C1,
-           CAST_SPEED7, DAM_UNDEFINED, SKILL_SPHERE_SUMMON, 5, true, nullptr);
+           CAST_SPEED3, DAM_ALIGN, SKILL_SPHERE_SUMMON, 5, true, nullptr);
```

Comment:
What does DAM_ALIGN do in this case?

Reply:
It stops the quickchant function from reducing the casting time of this spell further than it should.  It's a sloppy hack, but it's the only thing I could figure out short of completely rewriting the quickchant function.